text_generation/causal_lm/cpp/continuous_batching/python/tests/common.py

```diff
@@ -33,6 +40,15 @@ def get_beam_search() -> GenerationConfig:
     generation_config.num_return_sequences = generation_config.num_groups * generation_config.group_size
     return generation_config
 
+def get_beam_search_min_and_max_tokens() -> GenerationConfig:
+    generation_config = GenerationConfig()
+    generation_config.num_groups = 3
+    generation_config.group_size = 2
+    generation_config.min_new_tokens = 15
```

Comment:
to test such scenario we need special prompt, which will stop earlier than 15 tokens.. 

do we have such one?

Reply:
`What is OpenVINO?` :)